assets/agenda/components/AgendaCoverages.tsx

```diff
@@ -45,63 +48,89 @@ function AgendaCoveragesComponent({item, coverages, wireItems, actions, user, on
                     ))}
                 </div>
             )}
-            {coveragesWithState.map((coverage: any) => (
-                <div
-                    className={classNames(
-                        'coverage-item',
-                        {'coverage-item--clickable': onClick}
-                    )}
-                    key={coverage.coverage_id}
-                    onClick={onClick}
-                    title={onClick ? gettext('Open {{agenda}} in a new tab', window.sectionNames) : onClick}
-                >
+            {coveragesWithState.map((coverage: ICoverage) => {
+                const assigneeName = getCoverageAsigneeName(coverage);
+                const deskName = getCoverageDeskName(coverage);
+                const assignedUserEmail = coverage.assigned_user_email;
+                const assignedDeskEmail = coverage.assigned_desk_email;
+                const subject = `Coverage inquiry from NewsPro user: ${item.name || item.slugline}`;
```

Comment:
`gettext` must be used to flag this for translation.

Also `NewsPro` is a CP specific name for Newsroom. We should use the configured [SITE_NAME](https://github.com/superdesk/newsroom-core/blob/develop/newsroom/web/default_settings.py#L182) instead.

This config needs to be passed to the front-end, which can be done in the [scripts.html](https://github.com/superdesk/newsroom-core/blob/develop/newsroom/templates/scripts.html#L2) file, by adding something like `window.siteName = '{{ config.SITE_NAME }}';`.
Don't forget to add `siteName` to the [globals.d.ts](https://github.com/superdesk/newsroom-core/blob/develop/assets/globals.d.ts) file